lib/hanami/commands/server.rb

```diff
@@ -24,12 +27,15 @@ class Server < ::Rack::Server
       # @since 0.1.0
       # @see Hanami::Environment#initialize
       def initialize(options)
+        @entr_options =  options.merge(code_reloading: false).inject([]) {|res, (k, v)| res << "--#{k}=#{v}" ; res}.join(" ")
```

Comment:
yes, all passed CLI args will be forwarded to `entr` but with disabled code reloading